cantor-archive/src/main/java/com/salesforce/cantor/archive/file/AbstractBaseArchiverOnFile.java

```diff
@@ -19,6 +19,8 @@
 import static com.salesforce.cantor.common.CommonPreconditions.checkString;
 
 public abstract class AbstractBaseArchiverOnFile {
+    protected static final String FLAG_RESTORED = ".cantor-archive-restored";
```

Comment:
This has been removed, but the change isn't reflected in this PR. Please ignore :)